tests/test_electric_source_sink.py

```diff
@@ -12,11 +12,21 @@
 
 
 class TestMILPElectricSourceSink(TestCase):
-    """Unit tests for the MILP test case of a source, a cable, a sink"""
-
     def test_source_sink(self):
-        """Test to verify if when a target it set that is more or less than
-        then max or min of the respective component it is capped. 1000"""
+        """
+        Tests for the MILP test case of a source, a cable, a sink.
```

Comment:
1. I do not understand this: "Tests for the MILP test case ..."  
2. Suggested wording "... that consist out of a source, a cable and a sink.

Reply:
done